File: Python/Conversor_Grados.py
def fahrenheit(celsius):
    return (celsius * 9.0 / 5.0) + 32

def kelvin(celsius):
    return celsius + 273.15

def main():
    try:
        celsius = float(input("Ingrese los grados Celsius: "))
        fah = fahrenheit(celsius)
        kel = kelvin(celsius)
        print(f"Fahrenheit: {fah:.2f} °F")
        print(f"Kelvin: {kel:.2f} K")
    except ValueError:
        print("Error: Ingrese un número válido.")

if __name__ == "__main__":
    main()

File: aJava/src/Conversor_Grados.java
import java.util.Scanner;

public class Conversor_Grados {
    
    public static double fahrenheit(double celsius) {
        return (celsius * 9.0 / 5.0) + 32;
    }
    
    public static double kelvin(double celsius) {
        return celsius + 273.15;
    }
    
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Ingrese los grados Celsius: ");
        try {
            double celsius = scanner.nextDouble();
            
            double fah = fahrenheit(celsius);
            double kel = kelvin(celsius);
            
            System.out.printf("Fahrenheit: %.2f °F%n", fah);
            System.out.printf("Kelvin: %.2f K%n", kel);
        } catch (Exception e) {
            System.out.println("Error: Ingrese un número válido.");
        } finally {
            scanner.close();
        }
    }
}